installer/scripts/tue-install-impl.bash

```diff
@@ -110,7 +110,8 @@ function tue-install-target
         if [ -f $install_file.yaml ]
         then
             tue-install-debug "Parsing $install_file.yaml"
```

Comment:
we should throw an error/warning if a target doesn't contain a install.bash/yaml.

Reply:
Nah, committing an empty directory is quite hard in git. So someone who does that should know better ;-)
I might even think of a usecase where one would put only a README.md in a target and not a install.bash/yaml to explain what should happen or why this installer cannot cover this item.

Reply:
In a case that the there is only a readme. The user should be informed in some way that it should follow the steps in the readme. So this should be in the installer or in the target.